cmd/pipeline/main.go

```diff
@@ -437,6 +440,18 @@ func main() {
 			statusChangeDurationMetric,
 			clusterTotalMetric,
 		),
+		PKEOnVsphere: vspherePKEDriver.MakeVspherePKEClusterCreator(
+			vspherePKEDriver.ClusterCreatorConfig{
+				OIDCIssuerURL:               config.Auth.OIDC.Issuer,
+				PipelineExternalURL:         externalBaseURL,
+				PipelineExternalURLInsecure: externalURLInsecure,
+			},
+			logrusLogger,
+			authdriver.NewOrganizationGetter(db),
+			secret.Store,
```

Comment:
Please try to use the common secret store if it's not absolutely necessary to use the global one.

Reply:
Common secret store doesn't implement required functions